package.json

```diff
@@ -45,7 +45,7 @@
     "eslint-plugin-react-native": "^4.0.0",
     "husky": "^7.0.4",
     "jest": "^27.5.1",
-    "jest-fetch-mock": "^3.0.3",
+    "jest-fetch-mock": "https://github.com/jefflau/jest-fetch-mock#800c40d",
```

Comment:
The published version misses this polyfill https://github.com/jefflau/jest-fetch-mock/blob/800c40d9c73455281901fe527fc26fb1165cc528/src/index.js#L13-L15, which causes a TypeError in tests when a request aborts due to timeout